src/js/createKeyboard/index.js

```diff
@@ -0,0 +1,25 @@
+import {helperMessage} from "../globalVariables";
+
+export default class CreateKeyboard {
+  constructor() {
+    this.main = document.createElement('main');
+    this.main.id = 'main';
+
+    this.textarea = document.createElement('textarea');
+    this.textarea.id = 'textarea';
+    this.textarea.autofocus = true;
+
+    this.keyboardContainer = document.createElement('div');
+    this.keyboardContainer.id = 'keyboard';
+
+    this.descriptionKeyboard = document.createElement('div');
+    this.descriptionKeyboard.innerHTML = helperMessage;
+    this.descriptionKeyboard.id = 'descriptionKeyboard';
+  }
+
+  init() {
+    const { main, textarea, keyboardContainer, descriptionKeyboard} = this;
+    document.body.append(main);
```

Comment:
Not actually good approach, you need to take element to append keyboard from arguments, like:
`const keyboard = new Keyboard(document.body)`
`constructor(element) {
  this.element = element;
}`
then:
` init() {
   this.element.append(main);
}`
in that way you will be able to use your class everywhere.